pkg/sources/gitlab/gitlab.go

```diff
@@ -10,6 +10,7 @@ import (
 	"sync"
 
 	"github.com/go-errors/errors"
+	gg "github.com/go-git/go-git/v5"
```

Comment:
**Nit**: I like `gogit` more than `gg` because it is easier to understand what it is without looking at the imports.

Reply:
ah fair enough.